css/css-color/t425-hsla-basic-a.xht

```diff
@@ -9,12 +9,13 @@
 		<meta name="assert" content="Test basic functioning of hsla() colors." />
 		<style type="text/css"><![CDATA[
 		html, body { background: white; }
-		#one { color: hsla(120, 100%, 70%, 1.0); }
-		#two { color: hsla(120, 100%, 50%, 0.6); }
+		div { width: 600px; height: 30px; }
+		#one { background-color: hsla(120, 100%, 70%, 1.0); }
+		#two { background-color: hsla(120, 100%, 50%, 0.6); }
 		]]></style>
 	</head>
 	<body>
-		<p id="one">This text should be light green (the same color as the line below).</p>
```

Comment:
You should probably leave a line of text explaining the test above the boxes (will need to update both test and ref).

Reply:
Fixed